lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,161 @@
+;(function (define, accessibleModal) {
+    'use strict';
+
+    define(['jquery', 'backbone', 'underscore', 'underscore.string',
+            'text!templates/dashboard/course.underscore'],
+        function ($, Backbone, _, _s, courseTemplate) {
+
+            if (_.isUndefined(_s)) {
+                _s = _.str;
+            }
+
+            var CourseView = Backbone.View.extend({
+                el: '<li class="course-item" />',
```

Comment:
I think it would be cleaner to use a tagName and className for this (http://backbonejs.org/#View-el), i.e.
```
tagName: 'li',
className: 'course-item',
```